docs/pages/management/export-audit-events/fluentd.mdx

```diff
@@ -4,26 +4,56 @@ description: Forwarding events with Fluentd and Teleport event handler
 videoBanner: HAqxs4rBv2c
 ---
 
-In this guide, we will explain how to:
+Fluentd is an open-source data collector for a unified logging layer. In this
+guide, we will:
 
 - Set up Teleport's event handler.
 - Forward events with Fluentd.
 
+This guide also serves as an explanation for the Teleport Event Handler plugin,
+using Fluentd as the target service. We'll create a local Docker container as a
+destinatation for the event handler:
+
+{/*
+ The Graph below is made with this mermaidjs logic:
+ graph LR
+subgraph workstation [fa:fa-laptop Workstation]
+    style workstation fill:#C0C0C0,stroke: #000000
+    direction LR
+    c(teleport-event-handler)
+    style c fill: #a379ff,color: #ffffff
+    subgraph Docker
+        style Docker fill:#0db7ed,stroke: #000000
+        d(fluentd)
+    end
+    c-->d
+end
+e(Teleport)
+style e fill: #844cff,color: #ffffff
+e-->c
+ */}
+
+![Fluentd example diagram](../../../img/management/fluentd-demo-dia.png)
+
+You can follow the parameters below for a local proof-of-concept demo, or use any
+of the additional installation instructions to configure the Teleport Event Hanndler
```

Comment:
Which additional instructions does this sentence refer to?

Reply:
There is definitely a better way to say this. Mostly I'm referring to the helm tabs throughout.

It seems like this guide was originally written with local testing in mind, and over time additional instructions have been piecemealed and tacked on without looking at the guide holistically. This is my first pass attempt at trying to explain that.